### PR TITLE
More feedback improvements for bombs

### PR DESCRIPTION
**Pull request by MaxIsJoe**
part of #6149 

CL: [Improvement] Added feedback sounds and messages to bombs and the remote signaler

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Weapons/PowerSink.cs [hunk 1]
```diff
@@ -156,5 +161,14 @@ public override void ReceiveSignal(SignalStrength strength, ISignalMessage messa
 		{
 			ToggleActivity();
 		}
+
+		private IEnumerator BeepBeep()
+		{
+			while (isCharging && gameObject != null)
+			{
+				SoundManager.PlayNetworkedAtPos(beepSound, gameObject.AssumedWorldPosServer());
+				yield return WaitFor.Seconds(2f);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Items/Weapons/PowerSink.cs:170]: would be nice if it slowly sped up the closer to exploding it was

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Weapons/PowerSink.cs:170]: I like that idea, gonna add it once im done with another PR

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Weapons/PowerSink.cs:170]: ok cool

**Comment by MaxIsJoe** [comment]
Done, review again pls

File: UnityProject/Assets/Scripts/Items/Weapons/BulkyExplosive.cs [hunk 2]
```diff
@@ -26,7 +26,12 @@ public override IEnumerator Countdown()
 			spriteHandler.SetSpriteSO(activeSpriteSO);
 			if (GUI != null) GUI.StartCoroutine(GUI.UpdateTimer());
 			StartCoroutine(BeepBeep());
-			yield return WaitFor.Seconds(timeToDetonate); //Delay is in milliseconds
+			currentCountdown = timeToDetonate;
+			while (currentCountdown > 0 || gameObject != null)
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Items/Weapons/BulkyExplosive.cs]: What's the purpose of null checking `gameObject` in this case?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Weapons/BulkyExplosive.cs]: if the bomb gets destroyed for whatever reason while the courotine is still active and the StopCourtine gets called late this prevents it from throwing a NRE randomly when the while loop is still active.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Weapons/BulkyExplosive.cs]: it's a rare case but still worth covering

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Items/Weapons/BulkyExplosive.cs]: Decrementing `currentCountdown` wouldn't throw an NRE though, right? If anything it'd be something in `Detonate()`, I imagine, which isn't protected by this null check. Can probably move it after the loop.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Weapons/BulkyExplosive.cs]: Done 👍

**Review by Bod9001 — APPROVED** [review]

**Comment by MaxIsJoe** [comment]
meeeerrrgggeee

**Pull request merged**